src/Mod/Draft/draftguitools/gui_draft2sketch.py

```diff
@@ -90,7 +90,8 @@ def proceed(self):
         for obj in sel:
             if obj.isDerivedFrom("Sketcher::SketchObject"):
                 allDraft = False
-            elif obj.isDerivedFrom("Part::Part2DObjectPython"):
+            elif (obj.isDerivedFrom("Part::Part2DObjectPython")
+                    or obj.isDerivedFrom("Part::Feature")):
```

Comment:
```py
elif (obj.isDerivedFrom("Part::Part2DObjectPython")
      or obj.isDerivedFrom("Part::Feature")):
```

By the way, I'm not sure why this function tests separately the `Part::Part2DObject` from the `Part::Feature`. Be careful that you are testing the possible cases. The first one is intended to process the Draft scripted objects, while the second one is mostly for bare objects created from imported geometry such as SVG or DXF imports.

Ultimately, both `Part2DObject` and `SketchObject` are instances of `Part::Feature`.
```
Part::Feature --> Part::Part2DObject --> Sketcher::SketchObject
```

Reply:
With this change, both the draft objects and those imported from an svg file are converted to a single sketch. With the previous code, if a draft object is added to the imported object list, several sketches are created.